src/main/scala/strawman/collection/mutable/ArrayDeque.scala

```diff
@@ -0,0 +1,491 @@
+package strawman
+package collection.mutable
+
+import scala._
+import scala.collection.{generic, mutable}
```

Comment:
We might perform a few internal changes until #134 and #78 are fixed.

In your case you want to extend `mutable.IndexedSeq[A] with mutable.IndexedSeqOps[A, ArrayDeque, ArrayDeque[A]] with mutable.IndexedOptimizedGrowableSeq[A]`. We have to create `mutable.IndexedSeq` (the mutable hierarchy has been a bit left behind…).